addOns/graphql/src/main/javahelp/org/zaproxy/addon/graphql/resources/help/contents/graphql.html

```diff
@@ -47,8 +64,6 @@ <H2>Command Line</H2>
 <li>-graphqlendurl &lt;url&gt;  : Sets the Endpoint URL</li>
 </ul>
 
-The definitions will be imported synchronously and any warnings will be displayed on the command line.
```

Comment:
Isn't this true? We might need to change the command line then, the import should be synchronous as well.

Reply:
At least when in command line mode, if there's GUI it should still be async.